psalm.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <psalm
-  errorLevel="7"
+  errorLevel="5"
```

Comment:
Why have you lowered the error level?

Reply:
Different tools use different ways to define strictness. In this case, Psalm uses lower levels to increase strictness.

https://psalm.dev/docs/running_psalm/error_levels/